app/renderer/js/utils/domain-util.js

```diff
@@ -225,7 +225,14 @@ class DomainUtil {
 	}
 
 	reloadDB() {
-		this.db = new JsonDB(app.getPath('userData') + '/domain.json', true, true);
+		const domainJsonPath = path.join(app.getPath('userData'), '/domain.json');
+		try {
+			const file = fs.readFileSync(domainJsonPath, 'utf8');
+			JSON.parse(file);
+		} catch (err) {
+			fs.unlinkSync(domainJsonPath);
```

Comment:
@cPhost I think we should catch the err here and log using the console.js to make the debugging easier.